tests/test-team/helpers/test-user-client.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  AdminCreateUserCommand,
+  AdminDeleteUserCommand,
+  AdminSetUserPasswordCommand,
+  CognitoIdentityProviderClient,
+} from '@aws-sdk/client-cognito-identity-provider';
+import { AmplifyAppSyncClient } from './amplify-appsync-client';
+
+export class TestUserClient {
+  private readonly cognitoClient = new CognitoIdentityProviderClient({
+    region: 'eu-west-2',
+  });
+
+  private readonly userPoolId: string;
+
+  constructor(amplifyOutputsPathPrefix = '../..') {
+    this.userPoolId = new AmplifyAppSyncClient(
+      amplifyOutputsPathPrefix
+    ).getUserPoolId();
+  }
+
+  async createTestUser(email: string, password: string) {
+    const res = await this.cognitoClient.send(
```

Comment:
Do we want to check if it exists first or delete it just in case?

Reply:
we could do, but i'm not sure if it will change the behaviour very much - if we are trying to create a user that already exists, that is a problem, as the tests should be self-contained and responsible for their own data. so we should be throwing an error, which is what will happen now anyway if we try to create a user if it already exists

Reply:
What if the tests fail and leave the user behind?  I was thinking along the lines of self-cleaning tests, i.e. clean up after the last run before doing the new one.